packages/big-design/src/components/Pagination/Pagination.tsx

```diff
@@ -0,0 +1,119 @@
+import { ArrowDropDownIcon, ChevronLeftIcon, ChevronRightIcon } from '@bigcommerce/big-design-icons';
+import React, { useEffect, useState } from 'react';
+
+import { MarginProps } from '../../mixins';
+import { Dropdown } from '../Dropdown';
+import { Flex } from '../Flex';
+
+import { StyledButton } from './styled';
+
+export interface PaginationProps extends MarginProps {
+  currentPage: number;
+  itemsPerPage: number;
+  itemsPerPageOptions: number[];
+  totalItems: number;
+  onPageChange(page: number): void;
+  onRangeChange(range: number): void;
```

Comment:
🤔Should this now be `onItemsPerPageChange`?

Reply:
Nice catch, TY